### PR TITLE
Add the static lifetime specifiers

### PR DESCRIPTION
**Pull request by weikengchen**
* **make sure that you have executed all the following process and no errors occur**
  - [x] make fmt
  - [x] make lint
  - [x] make test

* **The major changes of this PR**

Rust 1.63.0 complains about some missing specifiers. Although 1.64.0 no longer complains, we feel it is a good idea to add them.

* **The major impacts of this PR**
  - [ ] Impact WASM?
  - [ ] Impact Web3 API?
  - [ ] Impact mainnet data compatibility?

* **Extra documentations**

### CODE REVIEW

File: src/components/config/src/abci/mod.rs [hunk 1]
```diff
@@ -525,7 +525,7 @@ pub mod global_cfg {
     #[cfg(not(test))]
     #[cfg(target_os = "linux")]
     fn check_rollback(m: &ArgMatches, cfg: &BtmCfg) -> Result<()> {
-        const HINTS: &str = r#"    NOTE:
+        const HINTS: &'static str = r#"    NOTE:
```
Comment (weikengchen) [inline src/components/config/src/abci/mod.rs:528]:
```suggestion
        const HINTS: &str = r#"    NOTE:
```

File: src/components/contracts/modules/evm/src/utils.rs [hunk 2]
```diff
@@ -14,7 +14,7 @@ use crate::{runner::ActionRunner, system_contracts::SystemContracts, Config};
 pub fn deploy_contract<C: Config>(
     ctx: &Context,
     contracts: &SystemContracts,
-    bytecode_str: &str,
+    bytecode_str: &'static str,
```
Comment (weikengchen) [inline src/components/contracts/modules/evm/src/utils.rs:17]:
```suggestion
    bytecode_str: &str,
```

File: src/components/wasm/src/wasm.rs [hunk 3]
```diff
@@ -98,7 +98,8 @@ use {
 
 /// Constant defining the git commit hash and commit date of the commit this library was built
 /// against.
-const BUILD_ID: &str = concat!(env!("VERGEN_SHA_SHORT"), " ", env!("VERGEN_BUILD_DATE"));
+const BUILD_ID: &'static str =
```
Comment (weikengchen) [inline src/components/wasm/src/wasm.rs:101]:
```suggestion
const BUILD_ID: &str =
```

**Comment by ktmlm** [comment]
try to upgrade the rust toolchain used by wasm compilation ?

**Comment by weikengchen** [comment]
I think let us close this PR since `nightly` also does not complain about it

**Pull request closed without merging**